plugins/inputs/vsphere/README.MD

```diff
@@ -0,0 +1,141 @@
+# VMware vSphere Input Plugin
+
+The VMware vSphere plugin uses the vSphere API to gather metrics from multiple vCenter servers.
+ 
+* Clusters
+* Hosts
+* VMs
+* Data stores
+
+## Configuration
+
+```
+# Read metrics from one or many vCenters
+[[inputs.vsphere]]
+  ## List of vCenter URLs, including credentials.
+  ## <protocol>://<user>:<password>@<vcenter>/sdk
+  ## Note the "@" characted must be escaped as %40
+  ## e.g.
+  ##     https://administrator%40vsphere.local:password@vcenter.local/sdk
+  #
+  vcenters = [ "https://administrator%40vsphere.local:password@vcenter.local/sdk" ]
+  
+  ## gather metrics from Clusters (default: true)
+  # gather_clusters = true
+  # cluster_metrics = []
```

Comment:
Use an include/exclude pair like in the [docker input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/docker#configuration).  There are types that make this easy to add.

Reply:
Thanks! I will look into that.